### PR TITLE
Facepunch Wiki Presence

### PR DESCRIPTION
**Pull request by MatStef132**
Hello!
I have again made a rich presence, this time for facepunch wiki. It supports all the pages on all of the wikis.
Proof of working and settings here: https://imgur.com/a/TCVE9ZO

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
You could do more than just showing "Viewing" messages...

**Comment by MatStef132** [comment]
I know, but really what else could be put in here? You can only put viewing here because there's nothing more. Maybe you have an idea to what can be added?

**Comment by ririxidev** [comment]
You could add for example getting title on what site user is and showing it, showing on what guide they are etc.

**Comment by MatStef132** [comment]
It's showing on which wiki the user is and which page he is on.

**Comment by ririxidev** [comment]
I meant different thing, but fine.

**Review by ririxidev — DISMISSED** [review]

**Review by ririxidev — APPROVED** [review]

File: websites/F/Facepunch Wiki/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "id": "269787765120565248"
+  },
+  "url": "wiki.facepunch.com",
+  "description": {
+    "en": "A wiki for facepunch games, which includes s&box, Garry's Mod and Rust"
+  },
+  "service": "Facepunch Wiki",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/7J6xKBQ.png",
+  "thumbnail": "https://imgur.com/dkTbScQ.png",
```
Comment (Bas950) [inline websites/F/Facepunch Wiki/dist/metadata.json:14]: upload a better logo
and thumbnail should be a promotional banner or a screenshot of the website homepage

File: websites/F/Facepunch Wiki/presence.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return Math.floor(Date.now() / 1000);
+  }
+
+  if (window.location.pathname === "/") {
+    presenceData.details = await presence.getSetting("homepageMessage");
+    presenceData.startTimestamp = setTimestamp();
+  } else {
+    presenceData.details = await presence.getSetting("viewingMessage") + " " + document.title;
+    presenceData.startTimestamp = setTimestamp();
+  }
```
Comment (Bas950) [inline websites/F/Facepunch Wiki/presence.ts:20]: maybe add some more statuses?
and I would personally put the document.title then on presenceData.state

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
no response from author - closing. Feel free to open new pr after doing some changes

**Pull request closed without merging**